JoinColony/colonyCDapp
# feat: added dark mode, changes in some components

## Description

- Added dark mode to the app
- Color changes in components that needed it

## Testing

* Toggle dark/light mode from settings menu 
<img width="385" alt="image" src="https://github.com/user-attachments/assets/371d7ecc-b1ba-4de9-a2d5-d2145fe57b43">

* Check if the app automatically detects system preferences

Resolves https://github.com/JoinColony/colonyCDapp/issues/2810
